terraform/modules/open_id_connect/roles.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_iam_role" "github_actions_terraform_plan" {
+  name               = "github-actions-terraform-ci-plan-read-only"
+  assume_role_policy = data.aws_iam_policy_document.github_actions_terraform_plan_assume_role.json
+}
+
+resource "aws_iam_role" "github_actions_terraform_admin" {
+  name               = "github-actions-terraform-admin"
+  assume_role_policy = data.aws_iam_policy_document.github_actions_terraform_admin_assume_role.json
+}
```

Comment:
Assuming the list I put on the ticket is correct these actually only need to exist in the dev account

Reply:
Have added in some code to make these roles conditional to the test environment